app/src/main/java/io/github/droidkaigi/confsched2018/data/db/entity/mapper/Converters.kt

```diff
@@ -13,11 +13,12 @@ object Converters {
         if (value == null) {
             return null
         }
-        return LocalDateTime.ofInstant(Instant.ofEpochSecond(value), ZoneId.systemDefault())
+        return LocalDateTime.ofInstant(
+                Instant.ofEpochSecond(value), ZoneId.of("JST", ZoneId.SHORT_IDS))
     }
 
     @JvmStatic
     @TypeConverter
     fun dateToTimestamp(date: LocalDateTime?): Long? =
-            date?.toEpochSecond(ZoneOffset.ofHours(9))
+            date?.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS))!!.toEpochSecond()
```

Comment:
Probably you can do this 👍 

```
date!!.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS)).toEpochSecond()
```

Reply:
I am kotlin beginner . . .

The suggestion is that an exception will occur if date is null.
If you follow the original code, how about the following code?
```
date?.atZone(ZoneId.of("JST", ZoneId.SHORT_IDS))?.toEpochSecond()
```
Or should an exception occur?